src/main/webapp/css/main.css

```diff
@@ -104,6 +107,7 @@ p {
   padding: 14px 16px;
   background-color: #46494C;
   font-family: inherit;
+  font-family: 'Righteous', sans-serif;;
```

Comment:
I think (?) having these two lines are redundant, might want to take out the first thing so you're not setting it twice?